.github/workflows/uberjar-sdk.yml

```diff
@@ -0,0 +1,149 @@
+name: Build + Docker Uberjar for SDK
```

Comment:
This is copied over from `uberjar.yml` and 
1. strip out unused parts e.g. steps that only run on master
2. all triggers outside `workflow_call`
3. Simplify the artifact name to `metabase-ee-uberjar`
4. accepts a new input `image_name` previously the image name is determined from a branch name that triggers the workflow, but we want to use the tag name (which created within the workflow) instead.